src/libPMacc/PMaccConfig.cmake

```diff
@@ -207,6 +279,8 @@ if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Intel")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DBOOST_NO_VARIADIC_TEMPLATES")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DBOOST_NO_CXX11_VARIADIC_TEMPLATES")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DBOOST_NO_FENV_H")
+elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+    add_definitions(-DBOOST_NO_CXX11_SMART_PTR)
```

Comment:
huh, weird. can you post some details?

also, do you want to keep the same style as above? (Although we should probably use `add_definitions` in all cases...)